sdk/keyvault/azure-security-test-keyvault-jca/src/test/resources/logging.properties

```diff
@@ -0,0 +1,9 @@
+#  Add VM parameter to enable this configure file: -Djava.util.logging.config.file="src/test/resources/logging.properties"
+
+handlers = java.util.logging.ConsoleHandler
```

Comment:
We probably should add documentation on how to use this file either separately or via a few comments here. I don't know the difference between what the `java.util.logging.ConsoleHandler.level`, `.level` or `com.azure.security.keyvault.jca.level` properties do.

Reply:
Added a new commit to provide more information about this file.